### ISSUE TITLE
Color "mismatch" on the Jetson Orin 8GB while running the code.

### ISSUE THREAD

**Issue by shivaprasad9999** [xactai/SCALPR-01.5#12]
Running the code on the Jetson Orin 8GB getting the output of the first car as white color in the console  where as while running on Gamdias getting output as grey color and one more red color color is detecting as blue
![MicrosoftTeams-image (9)](https://github.com/xactai/SCALPR-01.5/assets/152860439/8d55f7b4-6340-4e92-9515-c0253714472b)
![MicrosoftTeams-image (8)](https://github.com/xactai/SCALPR-01.5/assets/152860439/3c0a3a5d-5237-44d3-ab3d-b17e0cdc8723)
. As per annotations it is grey color.

Attached images are of Jetson Orin.

**Comment by Qengineering** [comment]
The difference between _Grey_ and _White_ is very narrow. Some vehicles with silvery paint look almost white.
I've hardcoded the distinguish on L=131 (HSL). Above, it is white, below it is grey.
With varying the frame rate, you don't always have the same frame hitting the wall: ergo, it's not always the same colour measurement.
For now, I print the found HSL colour on a debug line on the console. Use command line argument --verbose=5.
Later on we can adapt the threshold, or make it variable.

**Comment by shivaprasad9999** [comment]
Tried with verbose-5 as the command line argument getting the same results while running on Jetson Orin 8GB 1st car as white color.
![MicrosoftTeams-image (13)](https://github.com/xactai/SCALPR-01.5/assets/152860439/3e406a46-6a79-447c-a5c0-42048e51a799)

**Comment by Qengineering** [comment]
Indeed. As you can see, the value of the L colour component was 131. Just the threshold! If it was measured 130 you got a grey car.
